arkane/gaussianTest.py

```diff
@@ -130,5 +130,25 @@ def testLoadEthyleneFromGaussianLog_G3(self):
         self.assertEqual(conformer.spinMultiplicity, 1)
         self.assertEqual(conformer.opticalIsomers, 1)
 
+    def testLoadSymmetryAndOptics(self):
+        """
+        Uses a Gaussian03 log file for oxygen (O2) to test that its
+        molecular degrees of freedom can be properly read.
+        """
+
+        log = GaussianLog(os.path.join(os.path.dirname(__file__),'data','oxygen.log'))
+        optical, symmetry = log.get_optical_isomers_and_symmetry_number()
+        self.assertEqual(optical,1)
+        self.assertEqual(symmetry,2)
+
```

Comment:
Could you also test `self.assertIsInstance(log, GaussianLog)`?

Reply:
Great idea! I made the assertIsInstance an entirely new test, using the `determine_QM_software` method